### PR TITLE
Pypi

### PR DESCRIPTION
**Pull request by GrayEye**
Lets review and merge to Main and test out out PyPi and Github asset publications

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -41,8 +41,10 @@
     },
     include_package_data=True,
     install_requires=INSTALL_REQUIRES,
+    name="cfgenvy",
     packages=find_packages("src"),
     package_dir={"": "src"},
     python_requires=">=3.7",
+    url="https://github.com/pennsignals/cfgenvy",
```
Comment (jlubken) [inline setup.py]: In pyproject.toml.

**Review by jlubken — CHANGES_REQUESTED** [review]
One change.

File: setup.py (hunk 1 above)
Comment (GrayEye) [reply setup.py]: FIxed

**Review by jlubken — COMMENTED** [review]
LGTM

**Review by jlubken — APPROVED** [review]

**Pull request merged**